.github/workflows/tests.yaml

```diff
@@ -0,0 +1,29 @@
+name: Tests
+on:
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        jsonnet-version: ["0.20.0"]
+        jsonnet-bundler-version: ["0.5.1"]
```

Comment:
By not declaring `name` explicit for the job, these versions can show up in the name. If multiple variants of this job are run at some point, it will be helpful to not declare `name` explicitly and omitting the matrix parameters for the jobs.

Reply:
There is no plan to run this against multiple jsonnet and jb versions, I'd prefer to not turn this into a matrix and keep it a variable at the beginning of the file.